### ISSUE TITLE
External USB Mic crashes listener program

### ISSUE THREAD

**Issue by gutteringHearthplace** [Abaan404/dotfiles#1]
Basically what it says on the tin. I've had this issue for a few months now. I'm using one of your earlier configs, but I did check and that particular code hasn't changed since I first installed. The crash only occurs when the -b flag is passed and one of the audio devices is a microphone- which doesn't seem to have data for the volume.

```
'NoneType' object has no attribute 'get'
  File "/home/[REDACTED]/.dotfiles/scripts/listeners.py", line 505, in brief
    "volume": int(device.get("volume").get("front-left").get("value_percent")[:-1]),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/[REDACTED]/.dotfiles/scripts/listeners.py", line 481, in read
    data = self.brief(data)
           ^^^^^^^^^^^^^^^^
  File "/home/[REDACTED]/.dotfiles/scripts/listeners.py", line 42, in execute
    return self.dispatch(self.read())
                         ^^^^^^^^^^^
  File "/home/[REDACTED]/.dotfiles/scripts/listeners.py", line 544, in <module>
    Audio().execute()
AttributeError: 'NoneType' object has no attribute 'get'

```

I'll probably be trying to fix this one on my own, but I am by no means a python expert- figured I'd let you know.

**Comment by gutteringHearthplace** [comment]
Found the source of the problem- the mic has mono input:

```
      "volume": {
        "mono": {
          "value": 52429,
          "value_percent": "80%",
          "db": "-5.81 dB"
        }
```

**Comment by gutteringHearthplace** [comment]
fixed with #2

**Comment by Abaan404** [comment]
Thanks! Did not expect anyone else to seriously use my dots since its pretty buggy atm lol (although I'm guessing you're on the last eww commit which is semi-stable), cheers
